CONTRIBUTING.md

```diff
@@ -35,24 +35,7 @@ python3 -c 'import secrets; print(secrets.token_hex(100))' > .credentials/SECRET
       It will correspond to [`nixpkgs`](https://github.com/nixos/nixpkgs).
       In the **Settings** tab on that repository, in the **Features** section, ensure that _Issues_ are enabled.
 
-2.  For your GitHub user, in **Developer Settings**, generate a new [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
```

Comment:
Why is this removed? Doesn't this make it impossible to e.g. create a GH issue in a dev environment?